docs/modules/json.md

```diff
@@ -0,0 +1,110 @@
+[< Back](.)
+
+json
+================
+
+The `json` module provides a simple interface to read and write [json files and strings](https://www.json.org/json-en.html).
+
+It contains the following classes:
+
+* [Json](#json)
+* [JsonOptions](#jsonoptions)
+* [JsonError](#jsonerror)
+
+## Json
+
+### Constructor
+
+#### `constructor parse(value:String, options:Num)`
```

Comment:
Is the average user expected to directly call this constructor, or use the static `parse` and `fromFile` methods?

Reply:
Normally the expected way is using the static methods since they are a factory pattern using default options.
The construct is the base for all the other static methods.